File: review.py
import random

destinations = [
    "Paris", "Nova York", "Tóquio", "Roma", "Londres",
    "São Paulo", "Rio de Janeiro", "Cairo", "Dubai", " Peru"
]

reviews = []

def review_destination():
    selected_destination = random.choice(destinations)

    print(f"\nDestino para avaliação: {selected_destination}")
    stars = int(input("Dê uma nota de 1 a 5 estrelas: "))
    comment = input("Escreva um comentário: ")

    reviews.append({
        "destino": selected_destination,
        "estrelas": stars,
        "comentario": comment
    })

    print("\nAvaliação registrada!")
    print(f"Destino: {selected_destination}")
    print(f"Estrelas: {'*' * stars}")
    print(f"Comentário: {comment}\n")

File: review.dart
import 'dart:io';

List<String> destinations = [
  "Paris", "Tóquio", "Nova York", "Roma", "Londres",
  "Rio de Janeiro", "Bahamas", "Dubai", "Cidade do Cabo", "São Paulo"
];

List<Map<String, dynamic>> reviews = [];

void addReview(String destination, int stars, String comment) {
  reviews.add({"destino": destination, "estrelas": stars, "comentario": comment});
}

void showReviews() {
print("\n Avaliações de Destinos:");
    for (var review in reviews) {
      print("- ${review["destino"]}: ${"⭐" * review["estrelas"]} - ${review["comentario"]}");
    }
  }
}

void reviewDestination() {
  print("\nDestinos disponíveis para avaliação:");
  for (int i = 0; i < destinations.length; i++) {
    print("${i + 1}. ${destinations[i]}");
  }

  stdout.write("\nEscolha um destino (1-10): ");
  int choice = int.parse(stdin.readLineSync()!);
  String selectedDestination = destinations[choice - 1];

  stdout.write("Quantas estrelas ele merece?(1-5): ");
  int stars = int.parse(stdin.readLineSync()!);

  stdout.write("Deixe um breve comentario: ");
  String comment = stdin.readLineSync()!;

  addReview(selectedDestination, stars, comment);
  showReviews();
}
